backend/src/nodes/image_util_nodes.py

```diff
@@ -353,12 +352,29 @@ def __init__(self):
         self.description = "Rotate an image."
         self.inputs = [
             ImageInput("Image"),
+            SliderInput(
+                "Rotation Angle",
+                default=0,
+                maximum=360,
+                step=0.1,
+                controls_step=1,
```

Comment:
Could we make the control step larger? It would be nice to have something like 45° increments when clicking "up" in the number input box.